pxb/jenkins/pxb24-single-platform-run.yml

```diff
@@ -0,0 +1,222 @@
+- job:
+    name: pxb24-single-platform-run
+    project-type: matrix
+    description: |
+        Do not edit this job through the web!
+    concurrent: true
+    parameters:
+    - string:
+        name: PXB24_REPO
+        default: https://github.com/percona/percona-xtrabackup
+        description: URL to PXB24 repository
+    - string:
+        name: PXB24_BRANCH
+        default: percona-xtrabackup-2.4.20
+        description: Tag/Branch for PXC repository
+    - choice:
+        name: JOB_CMAKE
+        choices:
+        - /usr/bin/cmake
+        description: path to cmake binary
+    - string:
+        name: CMAKE_OPTS
+        default:
+        description: cmake options
+    - string:
+        name: MAKE_OPTS
+        default:
+        description: make options, like VERBOSE=1
+    - string:
+        name: XBTR_ARGS
+        default: ""
+        description: "./run.sh options, for options like: -j N Run tests in N parallel processes, -T seconds, -x options  Extra options to pass to xtrabackup"
+    - label:
+        name: OS_LABEL
+        default: "min-xenial-x64"
+        description: 'Choose OS platform from from this list
+        [ min-centos-7-x64,
```

Comment:
Why it was reworked from docker back to usual instances?

Reply:
Dev team normally use single platform job to test any failure on PXB param/param-medium jobs, and  PXB param/param-medium jobs use usual instances to run the test. To make the test run environment same we have replaced docker with the usual instance in single platform job. 
https://jira.percona.com/browse/PXB-2261?focusedCommentId=271877&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-271877